packages/core/marine-worker/src/index.ts

```diff
@@ -101,6 +112,16 @@ const toExpose = {
     },
 };
```

Comment:
I think if you just do `as const` here it will be almost exactly the same as if you use this `ExposedInterface` monstrosity. But it will make the type readonly, which doesn't matter in this case, I think. Is there a reason to use `ExposedInterface` over `as const`? If so - I would appreciate if you add a comment so we don't forget and I can learn

Reply:
You can check it's usage. I think that the name is pretty descriptive - it gives you an interface for interacting with wrapped thread.

Reply:
I see how it's used but it's hard to see the difference between

`ExposedInterface<typeof toExpose>` and just `typeof toExpose`

it would be useful for me how `ExposedInterface` generic "function" changes the original `typeof toExpose`, because I can't spot the difference between them